pinot-common/src/main/java/org/apache/pinot/common/utils/ServiceStartableUtils.java

```diff
@@ -47,10 +47,12 @@ private ServiceStartableUtils() {
    */
   public static void applyClusterConfig(PinotConfiguration instanceConfig, String zkAddress, String clusterName,
       ServiceRole serviceRole) {
-
+    int zkClientConnectionTimeoutMs =
+        (int) instanceConfig.getProperty(CommonConstants.Helix.ZkClient.ZK_CLIENT_CONNECTION_TIMEOUT_MS_CONFIG,
```

Comment:
(minor) Shouldn't convert to `int`

Reply:
It's the zk method call using int not long.